src/Provider/OrderLabelProvider.php

```diff
@@ -26,7 +26,7 @@ public function provideLabels(int $orderId, array $labelIds = [])
 
         if (! empty($labels)) {
             foreach ($labels as &$label) {
-                if ('1' === $label['is_return']) {
+                if (array_key_exists('is_return', $label) && '1' === $label['is_return']) {
```

Comment:
Ik zou isset doen, want array_key_exists is veel duurder en als de waarde NULL is dan moet ie toch false geven.